eras/shelley/impl/src/Cardano/Ledger/Shelley/TxAuxData.hs

```diff
@@ -124,6 +110,7 @@ hashShelleyTxAuxData ::
   ShelleyTxAuxData era ->
   SafeHash EraIndependentTxAuxData
 hashShelleyTxAuxData = hashAnnotated
```

Comment:
Strange, this `hashShelleyTxAuxData` wasn't even used in ledger, I would have expected the `hashTxAuxData` to use it  (but instead  that one is redefining it's  implementation , which is the implementation of `hashAnnotated` :raised_eyebrow:  ) 
Don't see it used in other repos either, probably some accidental leftover!